ajax/libs/flowplayer/package.json

```diff
@@ -16,5 +16,18 @@
     "type": "git",
     "url": "https://github.com/flowplayer/flowplayer"
   },
-  "license": "GPLv3"
+  "author": {
+    "name": "Niklas Närhinen",
+    "url": "https://github.com/nnarhinen"
+  },
+  "license": "GPL-3.0",
+  "npmName": "flowplayer",
```

Comment:
There are more old versions on the GitHub, which I create an issue to add thosd version in the future.
Both npm and GitHub have `6.0.3` and newer.